src/external/token/interfaces/IERC20Issuance_Blacklist_v1.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Internal
+import {IERC20Issuance_v1} from "@ex/token/IERC20Issuance_v1.sol";
+
+/**
+ * @title   ERC20 Issuance Token with Blacklist Functionality.
```

Comment:
Adding another comment as my original one is being ignored (on accident I believe) :D

> Needs to be copied from contract again after the changes to update.

Reply:
I would have said I did this, but maybe there was another one. Done